systemtests/src/main/java/io/enmasse/systemtest/Environment.java

```diff
@@ -94,6 +95,12 @@ private Environment() {
         if (installType == EnmasseInstallType.OLM) {
             log.info(debugFormat, OLM_INSTALL_TYPE, olmInstallType.name());
         }
+        if (url.startsWith("https://api")) {
+            try {
+                this.kubernetesDomain = new URL(url).getHost().replace("api", "apps");
+            } catch (Exception ignored) {
```

Comment:
Probably worth printing this exception and at least wrap it in a runtime exeception as I think this would be fatal?

Reply:
ok let me fix that

Reply:
I added printing of stacktrace, it is not easy to throw exception in Envritonment

Reply:
use throw new RuntimeException(ex)